{{cookiecutter.project_repo}}/pyproject.toml

```diff
@@ -0,0 +1,58 @@
+[tool.poetry]
+name = "{{ cookiecutter.project_slug }}"
+version = "{{ cookiecutter.version }}"
+authors = [
+    "UK Hydrographic Office <datascienceandengineering@ukho.gov.uk>"
+]
+description = "Python wrapper for the C implementation of the Generic Sensor Format library."
+license = "MIT"
+keywords = [
+    "FIXME",
+]
+classifiers = [
+    "Development Status :: 3 - Alpha",
+    "Intended Audience :: Developers",
+    "Intended Audience :: Science/Research",
+    "License :: OSI Approved :: MIT License",
+    "Natural Language :: English",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.6",
+    "Programming Language :: Python :: 3.7",
+    "Programming Language :: Python :: 3.8"
+]
+
+homepage = "https://github.com/UKHO/{{ cookiecutter.project_slug }}"
+repository = "https://github.com/UKHO/{{ cookiecutter.project_slug }}"
+
+readme = "README.md"
+
+# Do not manually edit, use poetry add/remove (--dev) to manage dependencies
+[tool.poetry.dependencies]
+python = "^3.6"
+
+[tool.poetry.dev-dependencies]
+assertpy = "^1.0"
+bandit = "^1.6.2"
+black = "^19.10b0"
+bump2version = "^1.0.0"
+coverage = "^5.1"
+flake8-isort = "^3.0.0"
```

Comment:
can remove this line and add an explicit dependency on isort

Reply:
Done